src/lib/containers/UserCard.tsx

```diff
@@ -66,74 +67,37 @@ export const UserCard: React.FunctionComponent<UserCardProps> = (
   // We fetch the image right away in case it is an expiring presigned URL
   const imageURL = usePreFetchResource(preSignedURL)
 
+  const { data: profileAndImage } = useGetUserProfileWithProfilePic(
+    principalId!,
+    {
+      enabled: !!principalId,
+    },
+  )
```

Comment:
We already had this hook, which does everything the deleted effect (below) was doing, minus some sessionStorage manipulation, so let's just use it here.